src/services/AltitudeService.js

```diff
@@ -0,0 +1,13 @@
+import { loadBvvAltitude } from './provider/altitude.provider';
+
+export class AltitudeService {
+
+	constructor(altitudeProvider = loadBvvAltitude) {
+		this._altitudeProvider = altitudeProvider;
+	} 
+
+	getAltitudeProvider() {
+		return this._altitudeProvider;
+	} 
```

Comment:
The service should provide a method `getAltitude(coordinate)` which uses the provider internally. No need to expose the provider...